addons/website_slides/models/slide_slide.py

```diff
@@ -279,7 +279,7 @@ def _compute_slide_views(self):
         for slide in self:
             slide.slide_views = mapped_data.get(slide.id, 0)
 
-    @api.depends('slide_ids.sequence', 'slide_ids.slide_type', 'slide_ids.is_published', 'slide_ids.is_category')
+    @api.depends('channel_id.slide_ids.sequence', 'channel_id.slide_ids.slide_type', 'channel_id.slide_ids.is_published', 'channel_id.slide_ids.is_category')
```

Comment:
Why this change ?

Reply:
While deleting category/section from backend It throws foreign key constraint. So according to product owner we need default behaviour there. But currently to get slides statistics data relationship is establish using 'slide_ids' field which is related to category_id. So due to wrongly/intended behaviour mapping, it throws constraint error. Rather then fetching from new relationship I have used existing one which is already created using channel_id.